test/test_torch.py

```diff
@@ -132,6 +132,8 @@ def test_type_conversion_via_dtype_name(self):
             self.assertEqual(x.half().dtype, torch.float16)
             self.assertEqual(x.int().dtype, torch.int32)
             self.assertEqual(x.bfloat16().dtype, torch.bfloat16)
+            self.assertEqual(x.cfloat().dtype, torch.complex64)
```

Comment:
this test is not really robust for complex tensors. I think we should additionally check if the imag and real values are as expected ...